csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/schema/SchemaParser.scala

```diff
@@ -475,6 +475,12 @@ trait SchemaParser extends RegexParsers
    */
   lazy val positiveIntegerExpr: PackratParser[PositiveIntegerRule] = "PositiveIntegerExpr" ::= "positiveInteger" ^^^ PositiveIntegerRule()
 
+
```

Comment:
I would like to see these rules numbered by the EBNF rule number. Also if these are introduced in 1.1, then you need to introduce a way for rules to be version specific; You should really do that before adding any more 1.1 rules otherwise it will be hard later to go back and edit each one for the appropriate version.